Makefile

```diff
@@ -28,8 +28,8 @@ INSTALL_BIN ?= /usr/bin/install
 SHARD_BIN ?= ../../bin
 CRFLAGS ?= -Dpreview_mt
 
-SRC_SOURCES ::= $(shell find src -name '*.cr' 2>/dev/null)
-DOC_SOURCE   ::= src/**
+SRC_SOURCES := $(shell find src -name '*.cr' 2>/dev/null)
+DOC_SOURCE   := src/cli.cr
```

Comment:
Why this change?

Reply:
The glob catches all kinds of things, including directories 🤦
And we really only need one entry point. That should cover all code and makes sure it's required in a working order.

Reply:
I believe you're correct (didn't tested it myself) 👍